action.yaml

```diff
@@ -25,6 +25,6 @@ inputs:
 runs:
   ## Injects git auth credentials into the host using .insteadOf rules in
   ## global .gitconfig.
-  using: "node16"
+  using: "node18"
```

Comment:
I don't think `node18` is supported and we have to move to `node20` https://github.com/actions/runner/discussions/2704

> From here, we plan to skip over node 18 and hop on the node 20 LTS release